### ISSUE TITLE
Hypermobile tympanic membrane

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10225]
**Preferred term label:**

**Synonyms**
tympanic membrane hypermobility

**Definition (free text, please give PubMed ID)**
PMID: 32901364 24% of patients were found to have **hypermobile tympanic membranes**; this is lower than the incidence of 36–46% reported in type 1 Stickler Syndrome patients. Otology questionnaire responses suggested that **tympanic membrane hypermobility** was not acquired through infectionrelated trauma.
PMID: 11556853 Hypermobility was sometimes associated with thin, visibly hyperflaccid tympanic membranes, which is a common and otoscopically detectable sequela of chronic or recurrent otitis media and/or previous tympanostomy tubes. However, 6 (21%) of 28 affected subjects with hypermobility had completely normal-appearing tympanic membranes and no history of otitis media or previous tympanostomy tube insertions. Type II collagen is known to be present in the tympanic membrane and the ossicular joints and, therefore, hypermobility may be a sequela of otitis media, a direct result of the primary collagen defect, or a combination of both of these factors. 

**Parent term (use hpo.jax.org/app)**
Abnormality of the tympanic membrane HP:0040090

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:90653  Stickler syndrome type 1
Stickler syndrome type 2 ORPHA:90654

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000015
- label: Tympanic membrane hypermobility
- parents: HP:0040090
- pmid: PMID:11556853|PMID:32901364
- def: Abnormally increased ability to move of the tympanic membrane.
- comment: In Stickler syndrome, hypermobility is sometimes associated with thin, visibly hyperflaccid tympanic membranes, which is a common and otoscopically detectable sequela of chronic or recurrent otitis media and/or previous tympanostomy tubes. Hypermobility may also be found with normal appearance tympanic membranes.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10225
- synonyms:
